src/models/analytics/analytics-tables.ts

```diff
@@ -401,8 +404,32 @@ const analyticsTablesL10n: Record<TableKeys, AnalyticsTable> = {
     },
     tableKey: 'gridPinGroup',
     tableText: {
-      en: { singular: 'Group-Location', plural: 'Group-Locations' },
-      nl: { singular: 'Groep-Locatie', plural: 'Groep-Locaties' },
+      en: { singular: 'Location-group', plural: 'Location-groups' },
```

Comment:
I think this was right as it was, it's not a list of grids of type pinGroup, but a list of pinGroups that belong to a grid. That means that a pinGroup can occur multiple times in this table (if it is in multiple grids). Analogous, I think the gridPin table below should be about "group ports", not "port groups".

Reply:
Fixed